netstandard/ref/System.Threading.Tasks.cs

```diff
@@ -436,4 +436,47 @@ public partial class UnobservedTaskExceptionEventArgs : System.EventArgs
         public bool Observed { get { throw null; } }
         public void SetObserved() { }
     }
+    [System.Runtime.CompilerServices.AsyncMethodBuilderAttribute(typeof(System.Runtime.CompilerServices.AsyncValueTaskMethodBuilder))]
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ValueTask : System.IEquatable<System.Threading.Tasks.ValueTask>
+    {
+        public ValueTask(System.Threading.Tasks.Sources.IValueTaskSource source, short token) { throw null; }
```

Comment:
Why is token using `short` ?

Reply:
Because `short` is the largest type that won't increase the size of `ValueTask`/`ValueTask<T>` in common cases.

See https://github.com/dotnet/coreclr/pull/16618#issuecomment-369397268 for much more detailed explanation by @stephentoub.

Reply:
@marek-safar, is this good to go then?